tests/authentication_tests/test_interactive_authentication.py

```diff
@@ -279,6 +294,23 @@ def test_console_auth_with_whitespace(self, *mocks):
             "2fa code"
         )
 
+    @patch(
+        "__builtin__.raw_input",
```

Comment:
confused by how the 'shared module' is laid out. this kind of test feels like it should be part of that module and not toolkit?

Reply:
Well.... layout issue, which we will field separately, aside... this test is to verify the behaviour of the console_authentication module when dealing with an SSO site... So this seems totally kosher to me.

I am not sure I understand your point.

Reply:
ignore comment above. I think it was more about the separate module than the test. Sorry for the noise :(